pump/storage/storage.go

```diff
@@ -40,7 +40,10 @@ import (
 
 const (
 	maxTxnTimeoutSecond int64 = 600
-	chanSize                  = 1 << 20
+	// the channel to buffer binlog meta, pump will block write binlog request if the channel is full
+	chanSize = 1 << 20
+	// if pump takes a long time to write binlog, pump will display the binlog meta information (unit: Second)
+	slowWriteDuration = 1.0
```

Comment:
Threshold would be clearer, for example, a similar concept is MySQL's query time threshold.

Reply:
It's for debug, I really don't want to make it complicate. 😢